### ISSUE TITLE
weird code in UIWebView+AFNetworking.m

### ISSUE THREAD

**Issue by kelystor** [AFNetworking/AFNetworking#3533]
```
[strongSelf loadData:responseObject MIMEType:MIMEType textEncodingName:textEncodingName baseURL:[task.currentRequest URL]];

if ([strongSelf.delegate respondsToSelector:@selector(webViewDidStartLoad:)]) {
	[strongSelf.delegate webViewDidFinishLoad:strongSelf];
}
```

I think '@selector(webViewDidStartLoad:)' should be '@selector(webViewDidFinishLoad:)'

**Comment by kcharwood** [comment]
Yes that appears to be a bug! Would you like to put up a PR?

**Comment by tsheaff** [comment]
@kcharwood why does AFNetworking call this explicitly? It results in the delegate's `webViewDidFinishLoad:` being called twice for me.

**Comment by kcharwood** [comment]
Will be fixed in #3727
